utils/_context/_scenarios/__init__.py

```diff
@@ -63,6 +63,12 @@ class _Scenarios:
 
     profiling = ProfilingScenario("PROFILING")
 
+    asm_e2e = EndToEndScenario(
```

Comment:
What about using the default scenario? the only APPSEC env var configured here is `DD_IAST_SECURITY_CONTROLS_CONFIGURATION`, all others values will be the same.

Reply:
The default scenario has `DD_TRACE_COMPUTE_STATS=true` which is impacting the test I did.

Reply:
ah :) Could you add this in the scenario's doc ?

Reply:
Done! By the way, It seems as this scenario is not being executed here on github. Any idea why?

Reply:
Because you need to add it in `.github/workflows/end-to-end.yml`. Speaking of which, could you change its name, maybe APPSEC_NO_STATS ?

Reply:
I did what you said. But it is still not executed on the pipeline :(

Reply:
Indeed, I found the reason, see my request change. Sorry, I should have seen that on my first review 😊

Reply:
All good mate! I applied your request change. Thanks for the help

Reply:
it's now executed : https://github.com/DataDog/system-tests/actions/runs/12990672620/job/36226967654?pr=3812#step:14:58

😉